wicker/core/storage.py

```diff
@@ -215,64 +215,115 @@ class S3PathFactory:
                     - part-1-attempt-2345.parquet
     """
 
-    def __init__(self, s3_root_path: str = None) -> None:
-        s3_config = get_config().aws_s3_config
-        if s3_root_path is None:
-            s3_root_path = s3_config.s3_datasets_path
-        self.root_path = s3_root_path
-        self.store_concatenated_bytes_files_in_dataset = s3_config.store_concatenated_bytes_files_in_dataset
+    def __init__(self, root_path: str, store_concatenated_bytes_files_in_dataset: bool = False) -> None:
+        """Init the path factory.
+
+        Object to form the expected paths and return them to the user based of root path and storage bool.
+
+        Args:.
+            root_path (str): File system loc of the root of the wicker file structure.
+            store_concatenated_bytes_files_in_dataset (bool, optional): Whether to assume concat bytes files are stored.
```

Comment:
hmm, do we mean "Whether to assume concatenated bytes files are stored in dataset folder"?

Reply:
done